src/BrazeKit-dev.js

```diff
@@ -358,8 +359,9 @@ var constructor = function() {
         }
     }
 
-    function initForwarder(settings, service, testMode) {
+    function initForwarder(settings, service, testMode, customFlags) {
```

Comment:
unfortunately there's a lot of arguments in initforwarder that arent used, per [the criteo kit](https://github.com/mparticle-integrations/mparticle-javascript-integration-criteo/blob/master/src/CriteoEventForwarder.js#L43):


```suggestion
    function initForwarder(settings, service, testMode, trackerId, userAttributes, userIdentities, appVersion, appName, customFlags) {
```